_summaries/katharopoulos2020transformers_1.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: summary
+title: Summary
+paper: katharopoulos2020transformers
+# Please fill out info below
+author: kelseyball
+score: 9
+---
+
+# Core Idea
+Expressing self-attention using kernel feature maps can reduce its time and space complexity from quadratic to linear
```

Comment:
Another core component to their paper was re-expressing their transformer model as a RNN to achieve much greater inference speeds.